src/applications/appeals/10182/config/form.js

```diff
@@ -50,6 +51,16 @@ import {
 import manifest from '../manifest.json';
 import hearingType from '../pages/hearingType';
 
+// add custom focust for the contact info mailing address edit page
+const confirmContactInfoEditMailingAddress = {
+  ...contactInfo.confirmContactInfoEditMailingAddress,
+  scrollAndFocusTarget: () => {
+    // even when not having this focusElement call the h3 still gets focused, sooo not sure if I should include this
+    // or just leave it blank since it's the default behavior if scrollAndFocusTarget is set to an empty function
+    focusElement('#main h3');
+  },
+};
+
```

Comment:
@Mottie Curious about how you would want to do this. Its a quick and dirty fix right now and mostly just a working example of how I fixed the focus issue for that page.

Reply:
Setting `useCustomScrollAndFocus` in the `config/form` file shifts focus from the `H2` to the `H3`, but only on initial render. If you think it works better with this focus function, then let's include it!

Reply:
Looks good! Thanks!